docker/README.md

```diff
@@ -25,4 +24,119 @@ the table above):
 docker pull osgeo/grass-gis:<tag>
 ```
 
+## Provided images
+
+Choose the image and running method that best suits your needs based on
+the required features, available disk space, and whether you need GUI support.
+
+### Alpine-based image
+
+Image size: around 240 MB
+
+**Installation:**
+
+```bash
+docker pull osgeo/grass-gis:releasebranch_8_4-alpine
+```
+
+**Running:**
+
+```bash
+docker run -it --rm osgeo/grass-gis:releasebranch_8_4-alpine grass -c EPSG:4326 /tmp/grassproject
+```
+
+### Debian-based image
+
+Image size: around 1.20 GB
+
+**Installation:**
+
+```bash
+docker pull osgeo/grass-gis:releasebranch_8_4-debian
+```
+
+**Running:**
+
+```bash
+docker run -it --rm osgeo/grass-gis:releasebranch_8_4-debian grass -c EPSG:4326 /tmp/grassproject
+```
+
+### Ubuntu-based image
+
+Image size: around 550 MB
+
+**Installation:**
+
+```bash
+docker pull osgeo/grass-gis:releasebranch_8_4-ubuntu
+```
+
+**Running:**
+
+```bash
+docker run -it --rm osgeo/grass-gis:releasebranch_8_4-ubuntu grass -c EPSG:4326 /tmp/grassproject
+```
+
+### Ubuntu-based image with GUI
+
+Image size: around 1.35 GB
+
+**Installation:**
+
+```bash
+docker pull osgeo/grass-gis:releasebranch_8_4-ubuntu_wxgui
+```
+
+**Running:**
+
+```bash
+docker run -it --rm -e DISPLAY=$DISPLAY \
+       -v /tmp/.X11-unix:/tmp/.X11-unix \
+       osgeo/grass-gis:releasebranch_8_4-ubuntu_wxgui grass
+```
```

Comment:
Is it clear that it won't work for windows (when not running through a WSL)?

Reply:
We may add Windows usage instructions (I'd rely on others to suggest them).